### PR TITLE
LPS-158766 [Raylife AP] Create validation data coverage

### PR DESCRIPTION
**Pull request by EugenioAntunes**
Sub task : [LPS-158766](https://issues.liferay.com/browse/LPS-158766)

### CODE REVIEW

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:relevant

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 1]
```diff
@@ -56,6 +58,10 @@ const CoverageInfoForm = () => {
 
 	const {form} = state?.steps?.coverage;
 
+	const vehicleInfoForm = state.steps.vehicleInfo.form;
+
+	const stateInfoForm = state.steps.contactInfo.form.state as StateInfoTypes;
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:63]:
```suggestion
	const vehicleInfoForm = state?.steps?.vehicleInfo?.form;

	const stateInfoForm = state?.steps?.contactInfo?.form?.state as StateInfoTypes;
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx [hunk 2]
```diff
@@ -134,6 +111,210 @@ const CoverageInfoForm = () => {
...
+							}}
+							value={form.vehicles[formIndex].comprehensive}
+						>
+							{comprehensiveOptions[stateInfoForm].map(
+								(comprehensiveOption, index) => (
+									<ClaySelect.Option
+										className="font-weight-bold text-center text-primary"
+										key={index}
+										label={comprehensiveOption.label}
+										value={comprehensiveOption.label}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Coverage/CoverageInfoForm.tsx:232]:
```suggestion
										label={comprehensiveOption.label}
										value={comprehensiveOption.value}
```

**Review by joanagrego — APPROVED** [review]
Nice job! Just added some suggestions :)

**Review by silasbsi — APPROVED** [review]
LGTM :1st_place_medal: Thanks @EugenioAntunes

**Comment by silasbsi** [comment]
ci:test:sf

**Pull request closed without merging**